qtpy/QtCore.py

```diff
@@ -10,22 +10,19 @@
 Provides QtCore classes and functions.
 """
 
-import os
-
-from qtpy import QT_API
-from qtpy import PYQT5_API
-from qtpy import PYQT4_API
-from qtpy import PYSIDE_API
+from qtpy import PYQT5
+from qtpy import PYQT4
+from qtpy import PYSIDE
```

Comment:
I agree with your previous comment that this could maybe be combined into a single line